src/index.ts

```diff
@@ -40,6 +40,13 @@ router.get("/roles/:roleName", getGuildRoleByRoleNameHandler);
 router.get("/discord-members", getMembersInServerHandler);
 
 router.get("/member/:id", getGuildMemberDetailsHandler);
+router.patch("/guild/member", changeNickname);
```

Comment:
May I know where these routes are being used ?

Reply:
I rebased it , from develop it came